_pages/en_US/installing-boot9strap-(soundhax-sp).txt

```diff
@@ -0,0 +1,33 @@
+---
+title: "Installing boot9strap (Soundhax)"
+---
+
+{% include toc title="Table of Contents" %}
+
+{% capture technical_info %}
+<summary><em>Technical Details (optional)</em></summary>
+
+For technical details on the exploits that you will be using on this page, see [here](https://github.com/nedwill/soundhax) (Soundhax) and [here](https://github.com/TuxSH/universal-otherapp) (universal-otherapp).
+
+{% endcapture %}
+<details>{{ technical_info | markdownify }}</details>
+{: .notice--info}
+
+### Compatibility Notes
+
+This Soundhax method (when combined with universal-otherapp) is compatible with versions x.x.x-0 through 11.15.0-36 in all regions. If the two numbers before the region in the system version string is equal to or more than 37, Soundhax is patched and won't work. If the version is above 11.15.0, universal-otherapp is patched and won't work.
```

Comment:
hey the guide says the version is x.x.x what does that mean

Reply:
cart updates do not exist past 11.10 or so, so the line about universal-otherapp while technically true cannot actually happen (fortunately)

Reply:
good to know